src/Core/Services/OpenAPI/OpenApiDocumentor.cs

```diff
@@ -169,16 +170,16 @@ public void CreateDocument()
         /// "/EntityName"
         /// </example>
         /// <returns>All possible paths in the DAB engine's REST API endpoint.</returns>
-        private OpenApiPaths BuildPaths()
+        private OpenApiPaths BuildPaths(RuntimeConfig runtimeConfig)
```

Comment:
does it make sense to pass in the entire runtimeconfig object? from the usage in this function, seems like there are only two properties of `runtimeConfig` that are used here and those seem to fit better in function signature.